components/create_post/create_post.jsx

```diff
@@ -1081,8 +1066,8 @@ export default class CreatePost extends React.Component {
         );
     }
 
-    handleEnableSendButton(message, fileInfos) {
-        return message.trim().length !== 0 || fileInfos.length !== 0;
+    shouldEnableSendButton() {
+        return this.state.message.length !== 0 || this.props.draft.fileInfos.length !== 0;
```

Comment:
This still needs `trim()` to prevent enabling send button when user enters enters empty spaces.